h2o-py/h2o/plot_result.py

```diff
@@ -0,0 +1,16 @@
+# -*- encoding: utf-8 -*-
+
+class H2OPlotResult(object):
```

Comment:
can you please move this into a `h2o.plot` submodule: the root module is already overloaded, thanks.